### PR TITLE
[feat]:Added hover effect to Supporters Cards

### PR DESCRIPTION
**Pull request by coder12git**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Added hover effects to the Supporters cards on the website. This enhancement would improve the user experience and provide visual feedback when interacting with the cards.

* **What is the current behavior?** (You can also link to an open issue here)

Issue -> #412
Currently, the cards on the website do not exhibit any visual change when hovered over. This can make the user interface feel static and less engaging.

* **What is the new behavior (if this is a feature change)?**

When a user hovers over a card, it would display a subtle hover effect to indicate interactivity. This would include changing the background color, and adding a border.

https://github.com/kubeedge/website/assets/108334168/82dca9f2-3b46-45c0-bd48-b286411784d1

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

No changes.

* **Other information**:
By implementing this hover effect, we can enhance the user experience and make the website's interface more dynamic and engaging.

### CODE REVIEW

**Comment by coder12git** [comment]
Hey @fisherxu please review my PR.

File: src/components/supporters/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import SectionContainer from "../sectionContainer";
 import Translate from "@docusaurus/Translate";
 import "./index.scss";
-
+import Link from "@docusaurus/Link";
 const supportList = [
```
Comment (Shelley-BaoYue) [inline src/components/supporters/index.js:6]: @coder12git  Thank you for your contributions!  I think adding a blank line here between import and following parts would make the format look better.

Comment (coder12git) [reply src/components/supporters/index.js:7]: > @coder12git Thank you for your contributions! I think adding a blank line here between import and following parts would make the format look better.

Done @Shelley-BaoYue

**Comment by coder12git** [comment]
Also @Shelley-BaoYue I want to inform you that I am looking forwared to contribute to this issue - #347, under LFX Fall'23. I have already submitted my application for this. I would love to get involved with this project and make and look forward to making an amazing contribution!

**Review by fisherxu — COMMENTED** [review]
@coder12git  Please squash the commits to fix the failed DCO test, thanks.

**Comment by coder12git** [comment]
> @coder12git Please squash the commits to fix the failed DCO test, thanks.

Done

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @Shelley-BaoYue

**Pull request merged**